docs/celeborn_ratis_shell.md

```diff
@@ -22,6 +22,8 @@ license: |
 Celeborn uses Ratis to implement the HA function of the master, Celeborn directly introduces ratis-shell package into the project
 then it's convenient for Celeborn Admin to operate the master ratis service.
 
+Since celeborn-0.6.0, the ratis RESTful API is supported, which is more convenient to operate the ratis service, see details in the swagger: `http://<CELEBORN_HOST>:<CELEBORN_PORT>/swagger#/Ratis`.
```

Comment:
Seems it is enough for the docs, cc @RexXiong 

<img width="1481" alt="image" src="https://github.com/user-attachments/assets/96245a95-3392-4392-9d65-eaff46e2be24">